IFC4x3/Properties/m/MeasurementEndTime_2h0TtkB$f46PgH3YITllt2/DocProperty.xml

```diff
@@ -0,0 +1,2 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
How about using the existing `StartDate` and `EndDate`? Of course making their descriptions a bit more generic.